test-framework/oauth-nimbus-poc/src/main/java/org/keycloak/test/framework/oauth/nimbus/OAuthClient.java

```diff
@@ -27,6 +28,7 @@
 import org.keycloak.test.framework.realm.ClientConfigBuilder;
 import org.keycloak.test.framework.realm.ManagedClient;
 import org.keycloak.test.framework.realm.ManagedRealm;
+import org.keycloak.test.framework.realm.ManagedUser;
```

Comment:
This is probably unused.

Reply:
Yer, added `resourceOwnerCredentialGrant(ManagedUser ..)` initially, but changed it, so no longer needed.

Removing